sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
What's the timeline of release sdk?

Reply:
I'm waiting for the confirmation from service team, now I'm not sure if he only needs the code ready or if he wants to release the package.